sdk/nodejs/admissionregistration/v1beta1/MutatingWebhookConfiguration.ts

```diff
@@ -95,6 +95,13 @@ import { getVersion } from "../../version";
               opts.version = getVersion();
           }
 
-          super(MutatingWebhookConfiguration.__pulumiType, name, props, opts);
+          const _opts = pulumi.mergeOptions(opts, {
+              aliases: [
+                  { parent: opts.parent, type: "kubernetes:admissionregistration.k8s.io/v1:MutatingWebhookConfiguration", name: name },
```

Comment:
Do you know if in general Kubernetes allows going "backwards"?  If it does, I suppose we may as well support it here too - but we typically only add aliases in the backward-compatible direction.

Reply:
Yes, I believe this is correct. Kubernetes stores the configuration in a common format and then converts to the specified apiVersion for each request.